### PR TITLE
Wire up new process actions to new process initializers

### PR DESCRIPTION
**Pull request by kerzhner**

### CODE REVIEW

File: packages/wallet/src/redux/reducer.ts [hunk 1]
```diff
@@ -87,30 +89,53 @@ function updatedState(
...
         processId,
-        action.channelId,
         action.playerIndex,
       );
-
-      return startProcess(state, sharedData, action, protocolState, processId);
-    case actions.protocol.CONCLUDE_REQUESTED:
-    case actions.protocol.CREATE_CHALLENGE_REQUESTED:
+    case actions.protocol.CONCLUDE_REQUESTED: {
+      const { state: protocolState, storage: sharedData } = challengeProtocolReducer.initialize(
```
Comment (andrewgordstewart) [inline packages/wallet/src/redux/reducer.ts]: I think this is out of order

Comment (andrewgordstewart) [inline packages/wallet/src/redux/reducer.ts]:
```suggestion
      const { state: protocolState, storage: sharedData } = concludeProtocolReducer.initialize(
```

File: packages/wallet/src/redux/reducer.ts [hunk 2]
```diff
@@ -87,30 +89,53 @@ function updatedState(
...
+        channelId,
+        processId,
+        incomingSharedData,
+      );
+      return { protocolState, sharedData };
+    }
+    case actions.protocol.CREATE_CHALLENGE_REQUESTED: {
+      const { state: protocolState, storage: sharedData } = concludeProtocolReducer.initialize(
```
Comment (andrewgordstewart) [inline packages/wallet/src/redux/reducer.ts]:
```suggestion
      const { state: protocolState, storage: sharedData } = challengeProtocolReducer.initialize(
```

File: packages/wallet/src/redux/reducer.ts [hunk 3]
```diff
@@ -1,15 +1,17 @@
...
+import * as actions from './actions';
 import { accumulateSideEffects } from './outbox';
-import { initializationSuccess } from 'magmo-wallet-client/lib/wallet-events';
-import { NewProcessAction, isNewProcessAction, isProtocolAction } from './protocols/actions';
-import * as funding from './protocols/funding/reducer';
+import { clearOutbox } from './outbox/reducer';
 import { ProtocolState } from './protocols';
-import { WalletProtocol } from './types';
+import { isNewProcessAction, isProtocolAction, NewProcessAction } from './protocols/actions';
+import * as challengeProtocolReducer from './protocols/challenging/reducer';
```
[the remark this answers is not in the document]
Comment (kerzhner) [reply packages/wallet/src/redux/reducer.ts]: Thank you for the recommendation. Addressed in https://github.com/magmo/apps/pull/424/commits/fd78c230d2c79eb3cb6911e66e190021695a6f87

File: packages/wallet/src/redux/__tests__/initialized.test.ts [hunk 4]
```diff
@@ -4,7 +4,7 @@ import * as states from './../state';
 import * as actions from './../actions';
 import * as scenarios from './test-scenarios';
 import { PlayerIndex, WalletProtocol } from '../types';
-import * as Funding from '../protocols/funding/reducer';
+import * as fundProtocol from '../protocols/funding';
```
Comment (andrewgordstewart) [inline packages/wallet/src/redux/__tests__/initialized.test.ts]:
```suggestion
import * as fundingProtocol from '../protocols/funding';
```

Comment (kerzhner) [reply packages/wallet/src/redux/__tests__/initialized.test.ts]: I was toying around with a different naming convention: `fundProtocol`, `challengeProtocol`, etc. I take it you prefer not to adopt that convention?

Comment (andrewgordstewart) [reply packages/wallet/src/redux/__tests__/initialized.test.ts]: That seems fine!

**Pull request merged**